lib/src/components/NxTag/__tests__/NxTag.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+
+import { NxSelectableTag } from '../NxTag';
+
+describe('NxSelectableTag', function() {
```

Comment:
Still worthwhile to have tests for `NxTag` to verify its structure and className logic

Reply:
> Still worthwhile to have tests for `NxTag` to verify its structure and className logic

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/9b3e4e505841a6214165df6ec0c0d3d0459a1bbd